app/uk/gov/hmrc/servicedependencies/controller/model/Dependencies.scala

```diff
@@ -17,32 +17,25 @@
 package uk.gov.hmrc.servicedependencies.controller.model
 
 import org.joda.time.DateTime
```

Comment:
Should we be using this library, elsewhere we use java.time.LocalDateTime?

Reply:
This isn't something that has been introduced by my changes.